### ISSUE TITLE
Errors in the watch process kills gulp

### ISSUE THREAD

**Issue by jaclyntan** [indigotree/atlas#65]
Hello. I am attempting to use your boilerplate and for the most part it is great! However I think your gulpfile needs to be updated because all errors I make in sass kills the process immediately, meaning I have to restart gulp all over again...

Are there any plans in the works to update it anytime soon?

**Comment by crgeary** [comment]
Hey,

I can't replicate this. If I make an error with the SASS it continues to work once that error has been corrected. If you'd like, you're welcome to make a PR to fix the issue.

There are plans to replace Gulp entirely, and use Hugo Pipes, but we have been waiting on Netlify to update their build environment to support it. I think they may have done this now, so expect to see this soon :)

**Comment by jaclyntan** [comment]
I ran into this problem when I had multiple scss files being imported and compiled from the sass folder. I am also using some pretty complex mixins like modular-scale and include-media, and it could be a combination of this and the gulpfile, but I tested a few clones of the repo with different sass directories and only got the issue when I tried compiling multiple sass files/directories and it errored out and crashed the server process.

Anyway, I have made a few changes to the file which I'll send as a pull request. I think the most important thing was, the error handler didn't exit gracefully if there was an error. 

Thank you for your awesome work on this though! I really appreciate the effort you've put into it.
